merkato/merkato.py

```diff
@@ -25,23 +25,45 @@ def __init__(self, configuration, coin, base, spread, bid_reserved_balance, ask_
         self.distribution_strategy = 1
         self.spread = spread # i.e '.15
         self.profit_margin = profit_margin
-        # Create ladders from the bid and ask bidget here
+
+        # Exchanges have a maximum number of orders every user can place. Due
+        # to this, every Merkato has a balance of coins that are not currently
+        # allocated. As the price approaches unallocated regions, the reserves
+        # are deployed.
         self.bid_reserved_balance = bid_reserved_balance
         self.ask_reserved_balance = ask_reserved_balance
+
+        # The current sum of all partially filled orders
+        self.base_partials_balance = 0
+        self.quote_partials_balance = 0
+
         self.user_interface = user_interface
+
         exchange_class = get_relevant_exchange(configuration[EXCHANGE])
         self.exchange = exchange_class(configuration, coin=coin, base=base)
+
         merkato_does_exist = merkato_exists(UUID)
 
         if not merkato_does_exist:
-            print('new merkato')
+            self._debug(1, "Creating new Merkato")
+
+            # Remove all active orders
             self.cancelrange(ONE_SATOSHI, ONE_BITCOIN)
+
             total_pair_balances = self.exchange.get_balances()
-            print('total_pair_balances', total_pair_balances)
+
+            self._debug(100, 'total_pair_balances: ', total_pair_balances)
+
             allocated_pair_balances = get_allocated_pair_balances(configuration['exchange'], base, coin)
-            check_reserve_balances(total_pair_balances, allocated_pair_balances, coin_reserve=ask_reserved_balance, base_reserve=bid_reserved_balance)
+            funds_available = check_reserve_balances(total_pair_balances, allocated_pair_balances, coin_reserve=ask_reserved_balance, base_reserve=bid_reserved_balance)
+
+            if not funds_available:
+                # Don't create the Merkato. Maybe should raise?
```

Comment:
this is done in check_reserve_balances it will error and kill

Reply:
Do we really want to do that though?

Reply:
Yes, I think that is appropriate, at least for now.